src/coreclr/jit/llvmcodegen.cpp

```diff
@@ -552,10 +552,10 @@ Value* Llvm::consumeValue(GenTree* node, Type* targetLlvmType)
             return _builder.CreatePtrToInt(nodeValue, Type::getInt32Ty(_llvmContext));
         }
 
-        // i32* e.g symbols, to i8*
+        // all pointers are opaque
         if (nodeValue->getType()->isPointerTy() && targetLlvmType->isPointerTy())
```

Comment:
Can this just be deleted then? Since all pointer types are (at the moment at least, since we don't use address spaces) are, presumably, equal?

Reply:
yes, it can, thanks